Gopkg.toml

```diff
@@ -1,28 +1,28 @@
 [[constraint]]
   name = "github.com/fatih/camelcase"
-  version = "1.0.0"
+  version = "^1.0.0"
 
 [[constraint]]
   name = "github.com/leboncoin/structs"
   version = "v1.0.1_lbc"
 
-
 [[constraint]]
-  name = "github.com/faberia/goavro"
-  version = "2.3.0"
+  name = "github.com/linkedin/goavro"
+  version = "^2.7.2"
 
 [[constraint]]
-  branch = "master"
   name = "github.com/mitchellh/mapstructure"
+  source = "github.com/CountFenring/mapstructure"
```

Comment:
Why depending of a old fork of the repository ?

Reply:
Because it's necessary to avoid to break our tests. This fork returns a empty array when no value is given, and the original version of mapstructure return nil now.